frontend/src/providers/walletProvider.tsx

```diff
@@ -2,13 +2,21 @@
 import { createWeb3Modal, defaultWagmiConfig } from "@web3modal/wagmi/react";
 
 import { configureChains, WagmiConfig } from "wagmi";
-import { foundry, mainnet, sepolia } from "wagmi/chains";
+import {
+    foundry,
+    mainnet,
+    sepolia,
+    optimismSepolia,
+    optimism,
+} from "wagmi/chains";
 import { publicProvider } from "wagmi/providers/public";
 
 // select chain based on env var
 const chainId = parseInt(process.env.NEXT_PUBLIC_CHAIN_ID || "31337");
 const chain =
-    [foundry, mainnet, sepolia].find((c) => c.id == chainId) || foundry;
+    [foundry, mainnet, sepolia, optimismSepolia, optimism].find(
+        (c) => c.id == chainId,
+    ) || foundry;
```

Comment:
Maybe define a constant variable with all the supported chains (maybe called `chains`?) and use it here, as not to have this weird intentation?

Reply:
Done @guidanoli ! Thanks.